server/queries/class/postQuestionQuery.ts

```diff
@@ -0,0 +1,10 @@
+import { Question } from '../../models';
+
+const postQuestionQuery = async (classId: string, question: string) => {
+  await Question.create({
```

Comment:
Here you are just executing the query, try returning the query and await it in the controller.

Reply:
Good Job Mustafa!